Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -315,6 +315,24 @@ theorem _root_.AlgEquiv.fieldRange_eq_top (f : E ≃ₐ[F] K) :
 
 end Lattice
 
+section equivMap
+
+variable {F : Type*} [Field F] {E : Type*} [Field E] [Algebra F E]
+  {K : Type*} [Field K] [Algebra F K] (L : IntermediateField F E) (f : E →ₐ[F] K)
+
+theorem map_eq : L.map f = (f.comp L.val).fieldRange := toSubalgebra_injective <| by
+  rw [toSubalgebra_map, AlgHom.fieldRange_toSubalgebra, AlgHom.range_comp, range_val]
+
+/-- An intermediate field is isomorphic to its image under an algebra map
+(which is automatically injective) -/
```

Comment:
```suggestion
/-- An intermediate field is isomorphic to its image under an `AlgHom`
(which is automatically injective) -/
```
to avoid confusion with `algebraMap`